Assets/VRTK/Scripts/Controls/3D/VRTK_Button.cs

```diff
@@ -464,6 +482,11 @@ protected virtual bool ReachedActivationDistance()
             return (Vector3.Distance(transform.position, restingPosition) >= activationDistance);
         }
 
+        protected virtual bool ReachedDeactivationDistance()
+        {
+            return (Vector3.Distance(transform.position, restingPosition) <= Vector3.kEpsilon);
```

Comment:
This line always returns false because `transform.position` is modified by physics when hitting Play.
It's also more realistic to trigger `Released` event when Button's position becomes less than activation distance instead of when reaching resting position.
The code is modified and tested. it's working well.